contracts/protocol/libraries/logic/MarketplaceLogic.sol

```diff
@@ -499,10 +497,7 @@ library MarketplaceLogic {
                 bool isNToken = reservesData[underlyingAsset].xTokenAddress ==
```

Comment:
If front end always buy or bid the underlying asset, then we can remove this branch?
```
if (vars.xTokenAddress == address(0)) {
}
```

Reply:
for moonbirds I'll ask them